cmd/orch-tester/orch_tester.go

```diff
@@ -70,7 +93,22 @@ func main() {
 	gsBucket := flag.String("gsbucket", "", "Google storage bucket to store segments")
 	gsKey := flag.String("gskey", "", "Google Storage private key (in json format)")
 
-	flag.Parse()
+	// Embedded Broadcaster config
+	var bCfg broadcasterConfig
+	bCfg.network = flag.String("network", "arbitrum-one-rinkeby", "Network to connect to")
+	bCfg.ethUrl = flag.String("ethUrl", "https://rinkeby.arbitrum.io/rpc", "Ethereum node JSON-RPC URL")
+	bCfg.datadir = flag.String("datadir", "", "Directory that data is stored in")
+	bCfg.ethPassword = flag.String("ethPassword", "", "Password for existing Eth account address")
+	bCfg.maxTicketEV = flag.String("maxTicketEV", "3000000000000", "The maximum acceptable expected value for PM tickets")
+	bCfg.maxPricePerUnit = flag.Int("maxPricePerUnit", 0, "The maximum transcoding price (in wei) per 'pixelsPerUnit' a broadcaster is willing to accept. If not set explicitly, broadcaster is willing to accept ANY price")
+
+	// Config file
+	_ = flag.String("config", "", "Config file in the format 'key value', flags and env vars take precedence over the config file")
+	err := ff.Parse(flag.CommandLine, os.Args[1:],
+		ff.WithConfigFileFlag("config"),
+		ff.WithEnvVarPrefix("OT"),
```

Comment:
Remember to update our deployments in `livepeer-infra` to include this prefix in env vars as well

Reply:
Yes, I'll update the whole `livepeer-infra` as part of this GH Issue https://github.com/livepeer/stream-tester/issues/159